Bottom: b55291e1aed2456173713ac26ad5a6e672a0b32e
Top:    cc4f2781e3b9218fb0603765dda1c0f0ceaf3863
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-07 02:18:24 +0000

draw.h: Remove unused DrawHoles() prototype


---

diff --git a/src/draw.h b/src/draw.h
index e0688ee..bae0cd8 100644
--- a/src/draw.h
+++ b/src/draw.h
@@ -70,7 +70,6 @@ void DrawLayerGroup (int side, const BoxType *drawn_area);
 void DrawPaste (int side, const BoxType *drawn_area);
 void DrawSilk (HID_DRAW *hid_draw, int side, const BoxType *drawn_area);
 void DrawMask (int side, const BoxType *drawn_area);
-void DrawHoles (bool draw_plated, bool draw_unplated, const BoxType *drawn_area);
 void PrintAssembly (int side, const BoxType *drawn_area);
 
 /* TEMPORARY */
